compiler/test/compilable/test21177.d

```diff
@@ -32,13 +32,13 @@ void main()
         printf("%*m");
 
         char* a, b;
-        sscanf("salut poilu", "%a %m", a, b);
+        sscanf("salut poilu", "%m %m", a, b);
```

Comment:
This is also wrong, you can't have `%m` as a naked specifier.

See https://issues.dlang.org/show_bug.cgi?id=23249